src/pages/dmt5/index.js

```diff
@@ -88,12 +79,23 @@ const DMT5 = () => {
             <Flexibility />
             {/* TODO: add/revise this section when swap free trading design is ready */}
             {/* <SwapFreeTrading /> */}
-            <DBanner
-                background_pattern={
-                    is_mobile ? BackgroundPatternDMT5_mobile : BackgroundPatternDMT5
-                }
-                title={<Localize translate_text="Get into the DMT5 experience" />}
-                data={data}
+            <StaticQuery
+                query={graphql`
+                    query {
+                        deriv_platform: file(relativePath: { eq: "dmt5-banner.png" }) {
+                            ...fadeIn
+                        }
+                    }
+                `}
```

Comment:
We can keep query defined at the top of the file, then we replace the current query with `query={query}` to be more readable